### PR TITLE
nomad debug renamed to nomad operator debug

### PR DESCRIPTION
**Pull request by langmartin**
This includes the commits in #8462, it will be rebased after that PR is merged.

### CODE REVIEW

File: command/commands.go [hunk 1]
```diff
@@ -484,6 +484,11 @@ func Commands(metaPtr *Meta, agentUi cli.Ui) map[string]cli.CommandFactory {
 				Meta: meta,
 			}, nil
 		},
+		"operator debug": func() (cli.Command, error) {
```
Comment (notnoop) [inline command/commands.go:487]: Maybe a comment that this is an alias that left for compatibility reasons but without us documenting it anymore?

File: website/pages/docs/commands/operator/debug.mdx [hunk 2]
```diff
@@ -0,0 +1,120 @@
+---
+layout: docs
+page_title: 'Commands: debug'
```
Comment (drewbailey) [inline website/pages/docs/commands/operator/debug.mdx:3]:
```suggestion
page_title: 'Commands: operator debug'
```

Comment (drewbailey) [reply website/pages/docs/commands/operator/debug.mdx:3]: going off of what keyring had https://www.nomadproject.io/docs/commands/operator/keyring

File: website/pages/docs/commands/operator/debug.mdx [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+sidebar_title: debug
+description: |
+  Build an archive of debug data.
+---
+
+# Command: debug
+
+The `debug` command builds an archive containing Nomad cluster
```
Comment (drewbailey) [inline website/pages/docs/commands/operator/debug.mdx:11]:
```suggestion
The `operator debug` command builds an archive containing Nomad cluster
```

File: command/commands.go (hunk 1 above)
Comment (langmartin) [reply command/commands.go:488]: done

File: website/pages/docs/commands/operator/debug.mdx [hunk 4]
```diff
@@ -0,0 +1,120 @@
+---
+layout: docs
+page_title: 'Commands: operator debug'
+sidebar_title: debug
+description: |
+  Build an archive of debug data.
+---
+
+# Command: debug
```
Comment (drewbailey) [inline website/pages/docs/commands/operator/debug.mdx:9]:
```suggestion
# Command: operator debug
```

**Pull request merged**